components/features/article/header/default.jsx

```diff
@@ -6,13 +6,20 @@ import { useAppContext } from 'fusion:context';
 const Header = () => {
   /* retrieve global content and site properties from fusion's app context */
   const appContext = useAppContext();
-  const { globalContent, siteProperties } = appContext;
+  const {
+    globalContent,
+    outputType,
+    siteProperties,
+  } = appContext;
 
   return (
     <header className="container-fluid">
-      <span className="text-info">News from Demo!</span>
-      <h2 className="card-title">{globalContent.headlines && globalContent.headlines.basic}</h2>
-      <h5 className="card-text">{globalContent.subheadlines && globalContent.subheadlines.basic}</h5>
+      {
+        outputType !== 'amp' &&
```

Comment:
I'm wondering if part of the code here is misleading. We are having toggles of amp in `default.jsx` but in the amp output type we are setting `fallback` to false. So it will never even get to this toggle when outputType === amp. Is that correct?

Reply:
or is this more for reference for an earlier part of lab 16 before we introduce `fallback`?

Reply:
Yes-- this is from an earlier part of the instructions. If the person is working through the instructions in order for their own branch, they'll do something like this first and *then* add the fallback line.